.gitignore

```diff
@@ -15,6 +15,8 @@ cypress/reports
 cypress/mochawesome-report/assets
 *.log
 .env
+package-lock.json
```

Comment:
change: `package-lock.json` needs to stay in Git :) Some more info on why: https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json